integration/spark/src/main/scala/org/apache/spark/sql/secondaryindex/rdd/SecondaryIndexCreator.scala

```diff
@@ -371,11 +377,12 @@ object SecondaryIndexCreator {
 
         val loadMetadataDetails = SegmentStatusManager
           .readLoadMetadata(indexCarbonTable.getMetadataPath)
-          .filter(loadMetadataDetail => successSISegments.contains(loadMetadataDetail.getLoadName))
 
+        val loadMetadataDetail = loadMetadataDetails
```

Comment:
better to rename it, else it will be confusing

Reply:
removed as logic changed